src/babel/index.ts

```diff
@@ -131,6 +136,11 @@ export default function ({
             const importSpecifierStringLiteral = t.stringLiteral(
               importSpecifier
             );
+            const processPath = process.cwd();
+            const modulePath = require.resolve(importSpecifier);
+            const relativeModulePath = modulePath.includes(processPath)
+              ? modulePath.replace(processPath, '.')
+              : modulePath;
```

Comment:
We need to harden this, as we need to build a path that is relative to the processPath (or more correctly relative to the root folder), not just a subset. Currently if modulePath is not including processPath, we will return an absolute path but webpack moduleId will have a relative path so it will not match up

Reply:
Done